evm_test_runner/src/main.rs

```diff
@@ -99,9 +98,7 @@ async fn run() -> anyhow::Result<bool> {
         }
     };
 
-    let parsed_tests_path = parsed_tests_path
-        .map(Ok)
-        .unwrap_or_else(get_default_parsed_tests_path)?;
+    let parsed_tests_path = get_default_parsed_tests_path()?;
```

Comment:
Hmm yeah, I guess there never really was a need to use a different path for parsed tests.